src/test/java/com/cdancy/bitbucket/rest/features/InsightsApiLiveTest.java

```diff
@@ -0,0 +1,245 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cdancy.bitbucket.rest.features;
+
+import com.cdancy.bitbucket.rest.BaseBitbucketApiLiveTest;
+import com.cdancy.bitbucket.rest.GeneratedTestContents;
+import com.cdancy.bitbucket.rest.TestUtilities;
+import com.cdancy.bitbucket.rest.domain.branch.Branch;
+import com.cdancy.bitbucket.rest.domain.common.RequestStatus;
+import com.cdancy.bitbucket.rest.domain.insights.Annotation;
+import com.cdancy.bitbucket.rest.domain.insights.AnnotationsResponse;
+import com.cdancy.bitbucket.rest.domain.insights.InsightReport;
+import com.cdancy.bitbucket.rest.domain.insights.InsightReportData;
+import com.cdancy.bitbucket.rest.domain.insights.InsightReportPage;
+import com.cdancy.bitbucket.rest.options.CreateAnnotations;
+import com.cdancy.bitbucket.rest.options.CreateInsightReport;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Collections;
```

Comment:
:warning: Wrong lexicographical order for 'java.util.Collections' import. Should be before 'org.testng.annotations.Test'. (#QR181:39#)-CHECKSTYLE_CustomImportOrderCheck